vars/lookupSnapshot.groovy

```diff
@@ -56,61 +59,95 @@ def call(body) {
 }
 
 @NonCPS
-def handleDBCluster(client, config) {
-  def outputName = config.get('envVarName', 'SNAPSHOT_ID')
-  def sortBy = config.get('snapshot', 'latest')
+def getAllDBSnapshots(client, request) {
+    List<DBSnapshot> allSnapshots = []
```

Comment:
@tarunmenon95 since this is groovy you can use `def` instead of defining the object type explicitly and is more idiomatic groovy.

Reply:
Ah yeah okay thats good to know, like python in that sense then. 
- Removed any explicit type settings in favor of `def` calls.

Tested code change, still works as expected.
<img width="649" alt="Screen Shot 2023-08-14 at 4 50 29 pm" src="https://github.com/base2Services/ciinabox-pipelines/assets/64295670/d27ceebc-3b08-4d11-8aaa-1bb32f2911cc">